src/components/Modal.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const Modal = ({ handleClose, show, title, body, onAction }) => {
+  return (
+    <div className={show ? "modal display-block" : "modal display-none"}>
+      <div className="modal-main" onClick={handleClose}>
+        <button className="button-div modal-top">
+          <img
+            src="/images/x-icon.svg"
```

Comment:
Can we use the same x-icon that we use for the mobile nav menu?

Reply:
the x-icon in the mobile nav menu is css created and not svg if I am not wrong :)